pkg/jx/cmd/get_pipeline.go

```diff
@@ -65,36 +70,91 @@ func NewCmdGetPipeline(f Factory, in terminal.FileReader, out terminal.FileWrite
 
 // Run implements this command
 func (o *GetPipelineOptions) Run() error {
-	jenkins, err := o.JenkinsClient()
+
+	_, _, err := o.JXClient()
 	if err != nil {
 		return err
 	}
-	jobs, err := jenkins.GetJobs()
+
+	_, _, err = o.KubeClient()
```

Comment:
also an empty kuberntes client

Reply:
It is a way to check jx and k8s connections are ok. I followed the [same approach than other commands](https://github.com/jenkins-x/jx/blob/7a411fca6febc7fdcf6df234a6978faa7e8b5a45/pkg/jx/cmd/start_pipeline.go#L97-L104).